Swift/Tests/LogTest.swift

```diff
@@ -46,6 +48,43 @@ class LogTest: CBLTestCase {
                          maxRotateCount: Database.log.file.maxRotateCount)
     }
     
+    func writeOneKiloByteOfLog() {
+        /// add a second delay to skip the filesystem flush
+        Thread.sleep(forTimeInterval: 1.0)
```

Comment:
Sound like this should be moved to after `writeAllLogs("123456789")`?

Reply:
i will check this and get more context on this